notesserver/settings/dev.py

```diff
@@ -13,4 +13,14 @@
     }
 }
 
-LOGGING = get_logger_config(debug=DEBUG, dev_env=True, local_loglevel='DEBUG')
+LOG_SETTINGS = {
```

Comment:
I don't see how or where this is used. Can we kill it?  Why or why not?

Reply:
Ah good catch.  This was part of my iterating and I left it here by accident.

Reply:
removed